server/routes/index.js

```diff
@@ -0,0 +1,27 @@
+module.exports = [{
+  method: 'GET',
+  path: '/{param*}',
+  handler: {
+    file: './server/public/index.html'
+  },
+  config: {
+    cors: true
+  }
+  }, {
+  method: 'GET',
```

Comment:
Remove, the first route captures the functionality of the second 2 routes as well.